src/overlays/actors/ovl_En_Syateki_Man/z_en_syateki_man.h

```diff
@@ -4,49 +4,70 @@
 #include "global.h"
 #include "objects/object_shn/object_shn.h"
 
-
 struct EnSyatekiMan;
 
 typedef void (*EnSyatekiManActionFunc)(struct EnSyatekiMan*, PlayState*);
 
-#define ENSYATEKIMAN_GET_FF00(thisx) (((thisx)->params & 0xFF00) >> 8)
+#define EN_SYATEKI_MAN_GET_PATH(thisx) (((thisx)->params & 0xFF00) >> 8)
+
+typedef enum {
+    /* 0 */ SG_GAME_STATE_NONE,             // None of the states below apply.
+    /* 1 */ SG_GAME_STATE_RUNNING,          // The shooting game is in-progress.
+    /* 2 */ SG_GAME_STATE_EXPLAINING_RULES, // For the Town Shooting Gallery, this state is also used for explaining the current high score.
+    /* 3 */ SG_GAME_STATE_NOT_PLAYING,      // Either the player said "No" to playing, or they said "Yes" but don't have enough rupees.
+    /* 4 */ SG_GAME_STATE_ONE_MORE_GAME,    // The player failed to get a new high score (Town) and/or perfect score (Swamp and Town).
+    /* 5 */ SG_GAME_STATE_GIVING_BONUS,     // The player gets bonus points at the end of the Swamp game if they get a perfect score.
+    /* 6 */ SG_GAME_STATE_ENDED,            // The player got a new high score and/or perfect score (Town), or the game is over (Swamp).
+    /* 7 */ SG_GAME_STATE_MOVING_PLAYER,    // The player is automatically moving towards the spot to play the game.
+} ShootingGalleryGameState;
+
+typedef enum {
+    /*  0 */ SG_OCTO_STATE_SPAWNING,
+    /*  1 */ SG_OCTO_STATE_SPAWNED,
+    /* 70 */ SG_OCTO_STATE_INITIAL = 70,
+    /* 80 */ SG_OCTO_STATE_HIDING = 80,
+} ShootingGalleryOctorokState;
 
-typedef struct {
-    /* 0x00 */ s16 index;
-    /* 0x04 */ f32 x;
-    /* 0x08 */ f32 y;
-    /* 0x0C */ f32 z;
-    /* 0x10 */ s32 variable;
-} EnSyatekiManUnkStruct;
+typedef enum {
+    /* 0 */ SG_OCTO_HIT_TYPE_NONE,
+    /* 1 */ SG_OCTO_HIT_TYPE_RED,
+    /* 2 */ SG_OCTO_HIT_TYPE_BLUE,
+} ShootingGalleryoctorokHitType;
 
 typedef struct EnSyatekiMan {
     /* 0x0000 */ Actor actor;
```

Comment:
Can you remove the leading 0's from the offset comments.

Reply:
I left them because IIRC, there was a PR that was removing them all. But sure, I can take care of them right now.